packages/access-api/src/kvs/spaces.js

```diff
@@ -22,8 +22,8 @@ export class Spaces {
   }
 
   /**
-   * @param {import('@web3-storage/access/capabilities/types').VoucherRedeem} capability
-   * @param {Ucanto.Invocation<import('@web3-storage/access/capabilities/types').VoucherRedeem>} invocation
+   * @param {import('@web3-storage/capabilities/types').VoucherRedeem} capability
```

Comment:
Nit: Would be nice if we export all the types from the `@web3-storage/capabilities` directly so we won't have to keep adding `/types` everywhere.

Reply:
I think it might not be supported in our current setup due to how we mix .js files and .ts file. can figure it out later but I think I tried it yesterday and settled for this to get it working.